processing/SpaceCrashAlpha/Player.pde

```diff
@@ -15,14 +12,11 @@ class Player extends Being {
 
   Player() {
     super();
-    position = new PVector(0, 0);
-    velocity = new PVector(0, 0);
-    acceleration = new PVector(0, 0);
+    physicsModel = new PointMass(false);
```

Comment:
ideally `PointMass.immovable == false` by default. what's a better way to do that?